controllers/healthcheck.go

```diff
@@ -29,16 +30,16 @@ func HealthCheck(w http.ResponseWriter, r *http.Request) {
 
 	err := db.Exec("SELECT 1").Error
 	if err != nil {
-		log.Err(err).Msg("Could not ping database")
+		utils.LogErr(r, err, "Could not ping database")
```

Comment:
Use the `utils.LogErrAndRespond()` instead of logging and responding separately.